lib/maze/option/processor.rb

```diff
@@ -12,6 +12,8 @@ class << self
         # @param config [Configuration] MazeRunner configuration to populate
         # @param options [Hash] Parsed command line options
         def populate(config, options)
+          config.bind_address = options[Maze::Option::BIND_ADDRESS]
+          config.port = options[Maze::Option::PORT]
```

Comment:
Is it worth forcing this to be an integer? I guess WEBrick will crash if it's not

```suggestion
          config.port = Integer(options[Maze::Option::PORT])
```

Reply:
I don't think we need to.  Thanks to the default on the option (an integer), Optimist treats it as a must-be integer option and rejects anything else:

```
$ bundle exec maze-runner --port=asd
Optimist::CommandlineError: option 'port' needs an integer
```

Reply:
fancy ✨